linker/shared/src/main/scala/org/scalajs/linker/analyzer/Infos.scala

```diff
@@ -572,11 +581,26 @@ object Infos {
 
             case NewArray(typeRef, _) =>
               builder.maybeAddAccessedClassData(typeRef)
+
             case ArrayValue(typeRef, _) =>
               builder.maybeAddAccessedClassData(typeRef)
+
+            case ArraySelect(_, _) =>
+              /* In theory, we'd need to reach ArrayIndexOutOfBoundsException
+               * here (conditional on the semantics) by IR spec.
+               * However, since the exact *constructor* is not specified, this
+               * makes little sense.
+               * Instead, the Emitter simply requests the exception in its
+               * symbol requirements.
+               */
+
             case ClassOf(cls) =>
               builder.maybeAddAccessedClassData(cls)
 
+            case GetClass(_) =>
+              // `GetClass` instantiates a java.lang.Class.
+              builder.addInstantiatedClass(ClassClass, ObjectArgConstructorName)
```

Comment:
This should also be added to `ClassOf`, I believe.

(Not that it makes an actual difference, because there will *always* be a `GetClass` node in `java.lang.Object.toString()`, which is always reachable because `jl.Object` is `instantiated()` and `toString()` is exported.)